README.md

```diff
@@ -1,2 +1,137 @@
-# Budget
-A budget app build with rails, this is the capstone project for the Rails module, part of Microverse.org Fullstack program
+<a name="readme-top"></a>
+<div align="center">
+  <img src="./budget.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Budget on Rails</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget ](#-budget-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run tests](#run-tests)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Budget <a name="budget"></a>
+
+**Budget** is a budgeting app build with rails, this is the capstone project for the Rails module, part of Microverse.org Fullstack program
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
```

Comment:
- [ ] Kindly add the  Live version of your project to the README.

- This should be deployed using Render, Netlify, or GitHub Pages.
- The project should be deployed properly, and the home and detail page should work as expected.
- Kindly make sure a link to the deployed project is present in the README.